utils/compatibility/scrapers/ingress-nginx.py

```diff
@@ -0,0 +1,52 @@
+# scrapers/ingress_nginx.py
+
+import requests
+from bs4 import BeautifulSoup
+from collections import OrderedDict
+from utils import printError, update_compatibility_info
+
+
+def scrape():
```

Comment:
This is pretty classic "arrow-code", putting a whole lot of logic in nested if-statements that becomes hard to maintain.  The simple solve is to invert the logic, eg use stuff like:

```python
if response.status_code != 200:
  print_error("could not fetch url")
  return

...rest of logic below
```

Pretty sure the entire function can be re-organized along those lines and require at most one indentation level.

Reply:
Makes sense, I'll refactor this and the other scrapers